README.md

```diff
@@ -2,3 +2,5 @@
 fun sunday demo
 
 Is GitHub fun?
+
+Yes it is
```

Comment:
I agree with this line! I will explicitly approve above.

Reply:
Sounds good. Looking out for it!